mautic/mautic
# Change timeout to segment stats type detection

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Times ago we added Segments stats improvements https://github.com/mautic/mautic/pull/7285
Segment added/removed log was added time ago, then we cannot use it for old segments 
Then segments stats need decide If use added line (just based on lead_lists_leads) or added/removed/total line graphs based on lead_event_log.

These stats depends on lead_event_log table data, If first item added to lead_event_log is -10 seconds is > like first added contact in lead_lists_leads.

But we found in our instance, the delay between added to lead_lists_leads and added to lead_event_log  was 13 seconds. I don't know why, but this PR just increase this timeout



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Just check If segments stats looks OK
